frontend/src/app/globals.css

```
@@ -387,3 +387,290 @@
+  image-rendering: pixelated;
+}
+
+.mc-slot--active {
+  outline: 3px solid var(--mc-emerald);
+  outline-offset: -1px;
+  box-shadow: inset 3px 3px 0 rgba(0, 0, 0, 0.55), inset -3px -3px 0 rgba(255, 255, 255, 0.06), 0 0 16px rgba(52, 211, 153, 0.55);
+}
+
+/* Stack-count number, bottom-right like real item stacks */
+.mc-count {
+  font-family: "Minecraft", sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Minecraft" (font-family-name-quotes)

(font-family-name-quotes)